### DISCUSSION TITLE
I cannot drag the map around.

### DISCUSSION THREAD

**Discussion by Asamoahfrancis** [visgl/react-google-maps#268] [General]
The map is static and fix at one point, can anyone help me fix it?

**Comment by mrMetalWood** [comment]
That sounds like you are using the controlled props `center` and `zoom` rather than the uncontrolled `defaultCenter` and `defaultZoom`. Using the "default" ones should enable you to move the map.

**Reply by Asamoahfrancis** [reply]
Thank you very much it worked fined :)

**Comment by iyyanarCommuniti** [comment]
my data lat,lng is dynamic one so the center prop is suitable for the map for dynamic map movement, here I'm facing the problem is I can't drag the map

**Reply by iyyanarCommuniti** [reply]
Can anyone help me to resolve this issue?

**Reply by usefulthink** [reply]
when you use the `center` prop for the map, you also need to use events to track user-interactions on the map, for example a handler for `onCameraChanged` that updates the state for the `center`, `zoom` etc. properties when the user interacts with the map. See here: https://visgl.github.io/react-google-maps/docs/api-reference/components/map#controlled-and-uncontrolled-props

**Comment by iyyanarCommuniti** [comment]
Thanks it worked.

**Comment by tianxiaozhang1** [comment]
Thank you
